README.md

```diff
@@ -7,13 +7,12 @@ A Pokemon Discord bot inspired by the late Pokecord.
 ## Installation
 Pikamon is built on Go 1.14+, please make sure this version is used to build and test the application.
 
-Build the Pikamon binary using the make target `build` (`rebuild` if you already have built it before):
+Build the Pikamon bot binary using the make target `bot`:
 ```shell script
-make build
-make rebuild
+make bot
 ```
 
-Once it is built, then the bot is can be run at minimum with the `--token` or `-t` flag:
+Once it is built, then the bot can be run at minimum with the `--token` or `-t` flag:
```

Comment:
so the token is something that we should never ever commit to the repo, and is a one time thing in the discord console. In short, you have to ask me for our bot's token. But, I can link to the discord documentation on how to create a new bot.

Reply:
Um, either way. I guess the bigger discussion point then becomes if we want others to test their bot's changes prior to doing a PR (or maybe as part of the PR) then how do we do that?

Reply:
I'll distribute the token to them in a secure manner. I created a test bot and server to make this safer, so the prod token can live in the deployment only. If someone gets the dev token, I can just rotate it and no damage is done.

So this will be a manual stage of becoming a contributor. I can add a line about reaching out to an admin to acquire the token.

Reply:
I think that would be good

Reply:
Added.

Reply:
thank you